### PR TITLE
Updating how we display messaging to unsupported browsers

### PR DESCRIPTION
**Pull request by codyrancher**
Proposed changes
======
IE and Legacy Edge never actually displayed the messaging
because ember failed to bootstrap due to a 'define' is undefined
error.

This adds a script to our index.html which controls weather we'll
display the messaging or finish bootstrapping ember.

This also adds Chromium edge to the list of unsupported. We may
want to walk this back however.

Types of changes
======
- Bugfix (non-breaking change which fixes an issue)

Linked Issues
======
rancher/rancher#27021

Further comments
======
**IE**
image

**Legacy Edge**
Screen Shot 2020-07-01 at 2 31 54 PM

**Edge**
image

**Firefox**
Screen Shot 2020-07-01 at 2 36 34 PM

**Chrome**
Screen Shot 2020-07-01 at 2 37 18 PM

### CODE REVIEW

**Comment by codyrancher** [comment]
I've removed Chromium Edge and I've opened rancher/rancher#27802 to track the Edge investigation.

**Comment by vincent99** [comment]
Surely nobody is regularly testing it, but the old engine Edge (which is `Edge/`?) used to work fine.  If someone wants to test that it's basically functional I don't see a need to hard block it.  But if major/obvious stuff is broken then it's not worth putting much time into fixing.

The Chromium-based current Edge (`Edg/`?) should definitely work and should be allowed in like any other chrome variant.

The error message could use more [sad-ie.png](https://github.com/rancher/ui/blob/master/public/assets/images/sad-ie.png).

**Comment by westlywright** [comment]
@codyrancher when you're ready can you open a 2.4 pr as well? Ticket is targeted at 2.4.6.

**Comment by loganhz** [comment]
Just FYI, commented out some lines in UI code.

Then UI works in old Edge work.

https://github.com/loganhz/ui/commit/09340f2232c5c1b54be95c5dd07f7e806940a50a

File: app/ie/template.hbs [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<section class="text-center">
-  <br><br>
-  <div class="sad-ie"></div>
```
Comment (westlywright) [inline app/ie/template.hbs:3]: Can we add the sad ie element back in the new script?

**Comment by westlywright** [comment]
@loganhz Will investigate those changes. Need to find a replacement for url-regex otherwise links in description content wont work. Also not sure we want to lock the ember template compiler by copying it to vendor, we want the same compiler version the dist is using. Also doing it that way would load that lib twice wouldn't it?

**Comment by loganhz** [comment]
@westlywright I think you are right. I can not recall why we have to change it :(

**Comment by vincent99** [comment]
Isn't url-regex the thing you were looking at replacing anyway because of an unrelated problem @westlywright ?

> IE and Legacy Edge never actually displayed the messaging because ember failed to bootstrap due to a 'define' is undefined error.

FWIW this obviously broke at some point, but it did actually work and show the page in the 1.x days at least.  A separate standalone script is a much better way to do it though.

**Comment by codyrancher** [comment]
I should've been more accurate and said it hasn't worked at least since July 2019 since that's as far back as I tested.

**Pull request merged**